src/ripple/app/main/NodeStoreScheduler.cpp

```diff
@@ -47,11 +47,14 @@ void NodeStoreScheduler::onChildrenStopped ()
 
 void NodeStoreScheduler::scheduleTask (NodeStore::Task& task)
 {
-    ++m_taskCount;
-    m_jobQueue->addJob (
-        jtWRITE,
-        "NodeObject::store",
-        [this, &task] (Job&) { doTask(task); });
+    if (! isStopping())
+    {
+        ++m_taskCount;
+        m_jobQueue->addJob (
+            jtWRITE,
+            "NodeObject::store",
+            [this, &task] (Job&) { doTask(task); });
+    }
```

Comment:
I'm conflicted about this.  I don't know that scheduling tasks during shutdown is harmful, but it is easy to believe it is, **especially** if testing indicates that this is so.  However, if scheduling tasks during shutdown _is_ harmful, this will only greatly reduce it, not eliminate it.

`isStopping()` could go from false to true once we're inside the `if` and prior to adding the task.  To do better, we need to add a `mutex` such that the check-and-schedule becomes an atomic operation.  But I would hate to hit this with that big of a hammer if we don't have to.  And maybe simply reducing the chances of scheduling during shutdown is good enough?